pkg/service/replay/service.go

```diff
@@ -63,3 +63,8 @@ type Telemetry interface {
 type RequestEmulator interface {
 	SimulateRequest(ctx context.Context, appID uint64, tc *models.TestCase, testSetID string) (*models.HTTPResp, error)
 }
+type TestResult interface {
+	TestRunStatus(ctx context.Context, status bool, testSetID string)
+	MockName() string
```

Comment:
function name cannot be a noun , how can I know what MockName, MockFile does and what are they are doing inside TestResult.

Reply:
Try to see if you use the same TestUtilInstance and add functions to it and use them.